app/models/katello/glue/candlepin/pool.rb

```diff
@@ -76,6 +76,7 @@ def organization
       # to define instance variables
       # TODO: break up method
       # rubocop:disable MethodLength
+      # rubocop:disable CyclomaticComplexity
       def load_remote_data(attrs)
```

Comment:
Perhaps break it up into individual lazy_accessor calls? TBH, with scoped search, most of these attributes should be pulled into the host record, right? I would classify a lot of the candlepin glue as "tech debt."